builtin/logical/transit/path_keys.go

```diff
@@ -154,6 +154,10 @@ func (b *backend) pathPolicyRead(
 			"deletion_allowed":       p.DeletionAllowed,
 			"min_decryption_version": p.MinDecryptionVersion,
 			"latest_version":         p.LatestVersion,
+			"supports_encryption":    p.Type.EncryptionSupported(),
```

Comment:
I don't really know what I think here, but given `deletion_allowed` does this make more sense as `encryption_supported`? An obvious reason not to do that is that it will cause the values, if sorted, to not be in a block. The answer is probably "no", but I wanted to float it.